### DISCUSSION TITLE
Is it available on Termux?

### DISCUSSION THREAD

**Discussion by hmmuaa** [fable-compiler/Fable#3975] [Q&A] [answered]
Nodejs v16.14 in Termux on Android?

I've found @fable-org/fable-standalone
but got no idea how to use it...

**Comment by MangelMaxime** [comment, marked as answer]
Hello,

I am not sure what Termux is.

Regarding `@fable-org/fable-standalone` this is a low level API package that can be used by other to execute Fable from JavaScript. For example, it is used to power:

- @fable-org/fable-compiler-js : A CLI application which works against basics .fsx or .fsproj files
- [Fable REPL](https://fable.io/repl/): Fable working inside of your browser

**Reply by hmmuaa** [reply]
Hello!
As I read I think this `@fable-org/fable-standalone` is able to run without .net but just nodejs right?
Is it available to use in js code but not command line?

I have tried this after npm install
```js
import * as f from'@fable-org/fable-standalone'
const log=console.log
log(Object.keys(f))
```
but I think I didn't make it correctly.

**Reply by MangelMaxime** [reply]
Yes, you can use it from Node JS.

You can look at how [Fable REPL](https://github.com/fable-compiler/repl) use it or [fable-compiler-js](https://github.com/fable-compiler/Fable/tree/main/src/fable-compiler-js).

They are both written using Fable itself, but you can use JavaScript to call it if you prefer. However, the API is perhaps the not most ergnomic one from JavaScript.

**Reply by hmmuaa** [reply]
Thanks for reply
